src/vizier/services/agent/pem/pem_manager.h

```diff
@@ -53,18 +53,19 @@ class PEMManager : public Manager {
  protected:
   PEMManager() = delete;
   PEMManager(sole::uuid agent_id, std::string_view pod_name, std::string_view host_ip,
-             std::string_view nats_url, px::system::KernelVersion kernel_version)
+             std::string_view nats_url, px::system::KernelVersion kernel_version,
+             bool kernel_headers_installed)
```

Comment:
optional: Might be worth a `KernelInfo` struct that includes `KernelVersion kernel_version` and the `bool kernel_headers_installed`

Reply:
Updated in 18853b052cb2be07eefbc1375e82b4d7683d9982